test/verify/test_step_brcst_trans.cpp

```diff
@@ -0,0 +1,24 @@
+
+#include "verify_program.hpp"
+#include <migraphx/program.hpp>
+#include <migraphx/generate.hpp>
+#include <migraphx/make_op.hpp>
+
+struct test_step_brcst_trans : verify_program<test_step_brcst_trans>
```

Comment:
Name this `test_step_broadcast_transpose`.

Reply:
fixed